releasenotes/notes/fix-neutron-keepalived-pid-cleanup-5faae4cf78b07f89.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Update neutron container images to apply keepalived PID clean up fix
+    I95a004a3acbe6a9160a19053a37fc0dd2b1875a5.
```

Comment:
Can you make a link to https://review.opendev.org/c/openstack/neutron/+/940287 instead?